### ISSUE TITLE
Ability to validate data

### ISSUE THREAD

**Issue by razzeeyy** [Razzeeyy/godot-networked-instancing-example#3]
There needs to be a way for server to validate and fix the received data in case there is a malicious client.

Possibly provide a `validated` signal that will run before `replicated` signal and will recieve the `old_data` and `new_data` variable and can modify/write to `new_data` to fix any discrepancies.

Possible caveates: need to be avare of time passed since last packet, also need to process packets on a per client base (in example not blend together elapsed time for packets from different peers).

Also was asked by https://github.com/Razzeeyy/godot-networked-instancing-example/issues/1#issuecomment-543095002

**Comment by razzeeyy** [comment]
Added in version 0.4.
See the [cheating example](https://github.com/Razzeeyy/godot-networked-instancing-example/tree/master/examples/cheating).

TL;DR: https://github.com/Razzeeyy/godot-networked-instancing-example/blob/master/examples/cheating/avatar.gd#L32
